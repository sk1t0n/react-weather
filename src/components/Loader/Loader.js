import React from 'react';
import './Loader.css';

function Loader(props) {
  return <div className="lds-dual-ring"></div>;
}

export { Loader };
