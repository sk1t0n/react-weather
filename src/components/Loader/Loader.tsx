import React from 'react';
import './styles.css';

const Loader: React.FC = () => (
  <div className="lds-dual-ring"></div>
);

export { Loader };
