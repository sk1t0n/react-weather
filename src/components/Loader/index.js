import React from 'react';
import './styles.css';

function Loader(props) {
  return <div className="lds-dual-ring"></div>;
}

export { Loader };
