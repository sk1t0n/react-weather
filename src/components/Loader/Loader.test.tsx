import { render } from '@testing-library/react';
import { Loader } from './Loader';

describe('Loader component', () => {
  it('check the snapshot', () => {
    const { asFragment } = render(<Loader />);
    expect(asFragment()).toMatchSnapshot();
  });
});
