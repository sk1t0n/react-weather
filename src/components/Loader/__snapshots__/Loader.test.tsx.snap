// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader component check the snapshot 1`] = `
<DocumentFragment>
  <div
    class="lds-dual-ring"
  />
</DocumentFragment>
`;
